homeassistant/components/device_tracker/__init__.py

```diff
@@ -81,7 +82,9 @@ def setup(hass, config):
 
         return False
 
-    tracker = DeviceTracker(hass, device_scanner)
+    seconds = util.convert(config[DOMAIN].get(CONF_SECONDS), int)
```

Comment:
You can pass in 12 here as the third argument to have it return in case of an error. (instead of line 132-133). For readability might even be better to add `DEFAULT_CONF_SECONDS = 12` to the top of the page.